### ISSUE TITLE
ArgumentNullException when loading vrm with standard shader

### ISSUE THREAD

**Issue by Ikeiwa** [vrm-c/UniVRM#1236]
## Environments:

 - UniVRM version: `0.83.0`
 - Unity version: `Unity-2019.4.29f1`
 - OS: `Windows 10`
 
When I try to load a vrm using the standard shader at runtime in a build I get an error.
The vrm material has an albedo, metallic, occlusion and normal map.
Also it doesn't happen when testing in unity, only in an exported build.
And also this bug doesn't happen with UniVRM < 0.60.0 but the model is extremely metallic and smooth.

```ArgumentNullException: Value cannot be null.
Parameter name: shader
UnityEngine.Material..ctor (UnityEngine.Shader shader) (at <cce5cfcb94df490bb4123d733b1a61c2>:0)
VRMShaders.OcclusionMetallicRoughnessConverter.get_Importer () (at F:/UnityProjects/VTubeEvents/Assets/VRMShaders/GLTF/IO/Runtime/OcclusionMetallicRoughnessConverter.cs:31)
VRMShaders.OcclusionMetallicRoughnessConverter.Import (UnityEngine.Texture2D metallicRoughnessTexture, System.Single metallicFactor, System.Single roughnessFactor, UnityEngine.Texture2D occlusionTexture, System.Boolean isLegacySquaredRoughness) (at F:/UnityProjects/VTubeEvents/Assets/VRMShaders/GLTF/IO/Runtime/OcclusionMetallicRoughnessConverter.cs:53)
VRMShaders.TextureFactory+<GetTextureAsync>d__13.MoveNext () (at F:/UnityProjects/VTubeEvents/Assets/VRMShaders/GLTF/IO/Runtime/TextureFactory.cs:105)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () (at <eae584ce26bc40229c1b1aa476bfa589>:0)
UniGLTF.ImporterContext+<LoadTexturesAsync>d__34.MoveNext () (at F:/UnityProjects/VTubeEvents/Assets/UniGLTF/Runtime/UniGLTF/IO/ImporterContext.cs:232)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Runtime.CompilerServices.TaskAwaiter.GetResult () (at <eae584ce26bc40229c1b1aa476bfa589>:0)
UniGLTF.ImporterContext+<LoadAsync>d__30.MoveNext () (at F:/UnityProjects/VTubeEvents/Assets/UniGLTF/Runtime/UniGLTF/IO/ImporterContext.cs:108)
Rethrow as AggregateException: One or more errors occurred.
Rethrow as AggregateException: One or more errors occurred.
UniGLTF.ImporterContextExtensions.Load (UniGLTF.ImporterContext self) (at F:/UnityProjects/VTubeEvents/Assets/UniGLTF/Runtime/UniGLTF/IO/ImporterContextExtensions.cs:22)
AvatarManager.Load (System.String avatar) (at F:/UnityProjects/VTubeEvents/Assets/Scripts/AvatarManager.cs:73)
ListAvatars+<>c__DisplayClass4_1.<RefreshAvatars>b__1 () (at F:/UnityProjects/VTubeEvents/Assets/Scripts/ListAvatars.cs:58)
UnityEngine.Events.InvokableCall.Invoke () (at <cce5cfcb94df490bb4123d733b1a61c2>:0)
UnityEngine.Events.UnityEvent.Invoke () (at <cce5cfcb94df490bb4123d733b1a61c2>:0)
UnityEngine.UI.Button.Press () (at F:/UnityProjects/VTubeEvents/Library/PackageCache/com.unity.ugui@1.0.0/Runtime/UI/Core/Button.cs:68)
UnityEngine.UI.Button.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at F:/UnityProjects/VTubeEvents/Library/PackageCache/com.unity.ugui@1.0.0/Runtime/UI/Core/Button.cs:110)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at F:/UnityProjects/VTubeEvents/Library/PackageCache/com.unity.ugui@1.0.0/Runtime/EventSystem/ExecuteEvents.cs:50)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at F:/UnityProjects/VTubeEvents/Library/PackageCache/com.unity.ugui@1.0.0/Runtime/EventSystem/ExecuteEvents.cs:261)
UnityEngine.EventSystems.EventSystem:Update() (at F:/UnityProjects/VTubeEvents/Library/PackageCache/com.unity.ugui@1.0.0/Runtime/EventSystem/EventSystem.cs:377)

**Comment by ousttrue** [comment]
Thank you for reporting.
This is due to a build that does not include the UniVRM shader.

#1187 
Make sure your build includes the UniVRM shader.

UniVRM shaders can be found in `Assets/VRMShaders` .
* MToon
* Unlit

And there is a texture conversion shader for import / export.
https://github.com/vrm-c/UniVRM/tree/master/Assets/VRMShaders/GLTF/IO/Resources/UniGLTF

**Comment by Ikeiwa** [comment]
Ok so I had UniVRM shaders imported but I had to add UniGLTF/StandardMapImporter to the Preloaded shader list and now it works. But now I have the same problem as the old version where I don't have the same visual result in unity editor and build.

![image](https://user-images.githubusercontent.com/13661338/134740435-845e9820-8e19-4781-a5df-9baef86c3074.png)

**Comment by ousttrue** [comment]
This issue may be caused by a fix in # 388 (`v0.69`).
Was this model exported by UniVRM before `v0.69`?

**Comment by Santarh** [comment]
The discussion has stopped for a long time, so I will close it.
If you have any problems, please comment again.

**Comment by AetherVortexYT** [comment]
![image_2024-12-18_200832237](https://github.com/user-attachments/assets/2e298a81-1c7c-47d8-ac23-991317f50232)
![image_2024-12-18_201044893](https://github.com/user-attachments/assets/bb0e44fd-b963-4f0e-971e-30aaf8fa61e1)

You see by the photos I placed the textures on the avatar and I used their shader.. but it becomes whiteout.. or whatever you wanna call it. I tried searching online on how to fix this issue but it seems that I'm the only one who has ever run into this issue. So this bug or whatever seems kinda new.. is there any fix for this?
